File: 리트코드/344.py
class Solution:
    def reverseString(self, s: list[str]) -> None:
        """
        Do not return anything, modify s in-place instead.
        """
        p1 = 0
        p2 = len(s) - 1
        while p1 < p2:
            c = s[p1]
            s[p1] = s[p2]
            s[p2] = c
            p1 += 1
            p2 -= 1

File: 리트코드/344.cpp
#include <iostream>
#include <vector>
#include <ctype.h>
using namespace std;

class Solution {
public:
    void reverseString(vector<char>& s) {
        int len = s.size();
        for (int i = 0; i < s.size() / 2; i++) {
            char tmp;
            tmp = s[i];
            s[i] = s[len - 1 - i];
            s[len - 1 - i] = tmp;
        }
    }
};

int main() {
    Solution s;

}